src/github.com/mozilla-services/pushgo/simplepush/etcd_balancer.go

```diff
@@ -426,7 +426,7 @@ func (b *EtcdBalancer) Fetch() (peers *EtcdPeers, err error) {
 		b.metrics.Increment("balancer.fetch.error")
 		return nil, err
 	}
-	b.metrics.Increment("balancer.fetch.success")
+	b.metrics.IncrementByRate("balancer.fetch.success", 1, 0.1)
```

Comment:
Not sure if this needs a low sample rate...we're only polling etcd every 10 seconds by default, and updating every minute. Even with both rates reduced to a second, that seems like a trickle compared to the storm of delivery metrics.